bluesky_httpserver/authorization/api_access.py

```diff
@@ -25,7 +25,7 @@
       - type: object
         additionalProperties: false
         patternProperties:
-          "^[a-zA-Z_][0-9a-zA-Z_]*$":
+          "^[a-zA-Z_][0-9a-zA-Z_.@]*$":
```

Comment:
```suggestion
          "^[a-zA-Z_][0-9a-zA-Z_.@\-]*$":
```

Reply:
If username can be a full email, I would add dashes to the character set. Otherwise I don't see any issues with the change.

Reply:
Also add the `+` symbol.